profiles/default/flink/Dockerfile

```diff
@@ -21,3 +21,9 @@ RUN ln -fs /opt/flink/opt/flink-s3-fs-hadoop-*.jar /opt/flink/plugins/flink-s3-f
 COPY --from=build /app/build/libs/FlinkJob.jar /scripts/FlinkJob.jar
 # UDFs
 COPY data/ /data
+
+# Add the wait script to the image
+COPY --from=ghcr.io/ufoscout/docker-compose-wait:latest /wait /wait
+
+# Launch the wait tool
+CMD /wait
```

Comment:
If you want to wait for a docker compose service to be started, lets use `depends_on` in docker compose and add a health check for `service_healthy`.

Reply:
Done.